script/script.js

```diff
@@ -7,13 +7,362 @@ const navItemPortfolio = document.querySelector('#nav-item-portfolio');
+      projectTecnologiesUl2.appendChild(githubLi);
+      projectTecnologiesUl2.appendChild(rubyLi);
+      projectTecnologiesUl2.appendChild(bootstrapLi);
+
+      // Creating the buttons
+      const btnPopupSeeLive = document.createElement('button');
+      btnPopupSeeLive.className = 'btn-see-live works-item-project-action-button';
+      const btnPopupSeeLiveSpan = document.createElement('span');
+      btnPopupSeeLiveSpan.textContent = 'See Live';
+      const btnPopupSeeLiveImg = document.createElement('img');
+      btnPopupSeeLiveImg.src = './img/icon-GitHub.png';
+      btnPopupSeeLiveImg.alt = 'github';
+      btnPopupSeeLive.appendChild(btnPopupSeeLiveSpan);
+      btnPopupSeeLive.appendChild(btnPopupSeeLiveImg);
+
+      const btnPopupSeeSource = document.createElement('button');
+      btnPopupSeeSource.className = 'btn-see-source works-item-project-action-button';
+      const btnPopupSeeSourceSpan = document.createElement('span');
+      btnPopupSeeSourceSpan.textContent = 'See Source';
+      const btnPopupSeeSourceImg = document.createElement('img');
+      btnPopupSeeSourceImg.src = './img/Icon - Export.svg';
+      btnPopupSeeSourceImg.alt = 'export';
+      btnPopupSeeSource.appendChild(btnPopupSeeSourceSpan);
+      btnPopupSeeSource.appendChild(btnPopupSeeSourceImg);
+
+      // creating the <div> that holds the buttons and appending its children
+      const projectPopupAction = document.createElement('div');
+      projectPopupAction.className = 'project-popup-action';
+      projectPopupAction.appendChild(btnPopupSeeLive);
+      projectPopupAction.appendChild(btnPopupSeeSource);
+
+      // <div> thats holds the ul's and the action <div>
+      const projectPopupTecnologies = document.createElement('div');
+      projectPopupTecnologies.className = 'project-popup-tecnologies';
+      projectPopupTecnologies.appendChild(projectPopupAction);
+      projectPopupTecnologies.appendChild(projectTecnologiesUl2);
+      projectPopupTecnologies.appendChild(projectTecnologiesUl);
+
+      // creating the div popup details
+      const projectPopupDetails = document.createElement('div');
+      projectPopupDetails.className = 'projects-popup-details';
+      projectPopupDetails.appendChild(projectPopupDescriptionDiv);
+      projectPopupDetails.appendChild(projectPopupTecnologies);
+
+      // Creating the <div> that holds the img
+      const projectPopupImg = document.createElement('div');
+      projectPopupImg.className = 'popup-project-img works-item-image-1';
+
+      // Creating the main container that holdds the popup
+      projectsPopupDiv.className = 'projects-popup';
+      projectsPopupDiv.classList.toggle('active-popup');
+      projectsPopupDiv.appendChild(popupProjectHeaderDiv);
+      projectsPopupDiv.appendChild(projectPopupImg);
+      projectsPopupDiv.appendChild(projectPopupDetails);
+      projectCardsSection.appendChild(projectsPopupDiv);
+    }
+  });
+}
```

Comment:
- Hi. As mentioned in the requirement, each popup is supposed to have details about a specific card but from your UI all the projects have popups with details about only one project 

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/37901469/195714712-5228c599-429e-4871-883b-a89b2a48a5bf.png">

- [ ] please Make sure each popup has details about the clicked project
- [ ] Currently, the popup is not scrollable and it's overflowing the screen. kindly fix it
- [ ] Please to improve UI ill recommend adding a blur or a background color with opacity when the popup shows